integration_tests/tests/rate-table-athena-tests.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  startQueryExecutionCommand,
+  formattedQueryResults,
+} from "../helpers/athenaHelper";
+import fs from "fs";
+import path from "path";
+import csvjson from "csvtojson";
+import { resourcePrefix } from "../helpers/envHelper";
+const prefix = resourcePrefix();
+
+describe("\n Execute athena query to retrieve rate details\n", () => {
+  test("retrieved rate details should matches with rate csv uploaded in s3 config bucket ", async () => {
+    const databaseName = `${prefix}-calculations`;
+    const queryString = `SELECT * FROM \"btm_rate_tables\"`;
+    const queryId = await startQueryExecutionCommand(databaseName, queryString);
+    const queryResult = await formattedQueryResults(queryId);
+    const queryResultToString = JSON.stringify(queryResult)
+      .replace(/\s00:00:00.000/g, "")
+      .replace(/\b(\d+)(?:(\.\d*?[1-9]+)0*|\.0*)/g, "$1$2");
```

Comment:
the regex removes trailing zeros  after decimal   Initially i tried minimising decimal places but that didn't work because if csv has transaction_price as 1 then in athena table it is showing as 1.0

Reply:
Thanks removed the boundary now